Update instructions for adding or removing tags in a project (#1906)
semgrep/semgrep-docs: docs/semgrep-appsec-platform/tags.md

To manage tags through Semgrep AppSec Platform, follow these steps:

1. Go to the Semgrep AppSec Platform [Projects](https://semgrep.dev/orgs/-/projects) page.

> 2. Find the project to which you want to add or remove tags, then click its **<i class="far fa-window-restore"></i> icon** under **Details**.

Reviewer: ```suggestion
2. Find the project you want to modify, then click its **<i class="far fa-window-restore"></i> icon** under **Details**.
```

Reviewer: this is one of those places where both options for the prepositional phrase sound a bit weird (yours is very correct, but sounds formal, the other way leaves a hanging "to" that sounds strange), so I went for another phrasing entirely

3. Click the **Settings** tab.

> 4. Add or remove tags under **Tags** section and Click **Save**.

Reviewer 2: ```suggestion
4. Add or remove tags under the **Tags** section
5. Click **Save**.
```
People might not see "Click Save" at the end so I'm moving it to the next line.

> 4. Add or remove tags under **Tags** section and Click **Save**.

Reviewer 3: ```suggestion
4. In the **Tags** section, click to add tags from the drop-down menu or click the **X** to remove a tag.
5. Click **Save**.
```
